src/main/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionContext.java

```diff
@@ -116,6 +116,7 @@ public VP8AdaptiveTrackProjectionContext(
         logContext.put("id", Integer.toString(hashCode()));
         this.logger = parentLogger.createChildLogger(VP8AdaptiveTrackProjectionContext.class.getName(), logContext);
         this.payloadType = payloadType;
+        this.vp8QualityFilter = new VP8QualityFilter(parentLogger);
```

Comment:
Do we not want to pass `logger` here to keep the `id` we just added? Also `id` is generic and might clash with something from the parent's context, maybe rename it? I can't think of a good name though

Reply:
i'm not sure we actually even need the added 'id' here.  i added it a while back when i found it useful during debugging, but we'll have more context now.  i'm going to remove it and we can re-add it in the future if we need it.